r/R/RecordBatch.R

```diff
@@ -23,6 +23,7 @@
     num_rows = function() RecordBatch__num_rows(self),
     schema = function() `arrow::Schema`$new(RecordBatch__schema(self)),
     to_file = function(path) invisible(RecordBatch__to_file(self, fs::path_abs(path))),
+    to_raw = function() RecordBatch__to_raw(self),
```

Comment:
Maybe "to_stream_raw`?

Reply:
How about `to_stream()`, I think this implies that the returned data is `raw()`.